svc-bie-kafka/src/main/resources/application.yaml

```diff
@@ -1,4 +1,8 @@
 spring:
+  profiles:
+    # See https://github.com/department-of-veterans-affairs/abd-vro/wiki/Configuration-settings#vros-use-of-spring-profiles
+    active: ${ENV:default}
```

Comment:
Causes `application-$ENV.yaml` to be loaded. In LHDI, `ENV` is set to `dev`, `qa`, `prod-test`, ... -- see https://github.com/department-of-veterans-affairs/abd-vro/wiki/Development-environments